e2e/help/help.go

```diff
@@ -24,49 +24,54 @@ type ctx struct {
 	env e2e.TestEnv
 }
 
-var helpContentTests = []struct {
+var helpOciContentTests = []struct {
+	name string
 	cmds []string
 }{
 	// singularity oci
-	{[]string{"help", "oci"}},
-	{[]string{"help", "oci", "attach"}},
-	{[]string{"help", "oci", "create"}},
-	{[]string{"help", "oci", "delete"}},
-	{[]string{"help", "oci", "exec"}},
-	{[]string{"help", "oci", "kill"}},
-	{[]string{"help", "oci", "mount"}},
-	{[]string{"help", "oci", "pause"}},
-	{[]string{"help", "oci", "resume"}},
-	{[]string{"help", "oci", "run"}},
-	{[]string{"help", "oci", "start"}},
-	{[]string{"help", "oci", "state"}},
-	{[]string{"help", "oci", "umount"}},
-	{[]string{"help", "oci", "update"}},
+	{"HelpOci", []string{"help", "oci"}},
```

Comment:
you could get rid of `"help"` for all lines, and use `WithCommand("help"), WithArgs(tc.cmds...)` around https://github.com/sylabs/singularity/pull/3924/files#diff-2f7903baca57c225790dd52d14e1162eR60